spec/message.abnf

```diff
@@ -54,11 +54,13 @@ local = %s".local"
 match = %s".match"
 
 ; Reserve additional .keywords for use by future versions of this specification.
-reserved-statement = reserved-keyword [s reserved-body] 1*([s] expression)
+reserved-statement = reserved-keyword [s reserved-body-in-statement] 1*([s] expression)
```

Comment:
I think we should steer clear of the `-in-statement` part.

```abnf
reserved-statement = reserved-keyword [s reserved-body] 1*([s] expression)
reserved-body = reserved-body-start *([s] 1*reserved-body-part)
reserved-body-start = *([s] 1*reserved-body-part)
reserved-body-part = reserved-char / reserved-escape / quoted
```

I recognize that `reserved-body` is also used by `private-use-annotation`. However, the intention there should not be to capture ending whitespace either. `private-use-annotation` is part of annotation and has the same relationship to e.g. the optional `attribute` in `expression`. Trimming private use is thus also desirable to eliminate ambiguity.